newrelic/alert/outputs.tf

```diff
@@ -0,0 +1,43 @@
+output "newrelic_alert_channel_tfer--aidan-002D-feldman-002D-aidan-002D-feldman-002D-gsa-002D-gov-002D-2585255_id" {
```

Comment:
Are these outputs actually used anywhere? If not, let's exclude them to cut down the noise. Ditto for the other modules.